app/assets/stylesheets/collection.scss

```diff
@@ -0,0 +1,3 @@
+.collection-result {
+  padding: 10px 5px;
```

Comment:
I feel like there may be a variable that provides something similar to what we're trying to get at here, but not sure what that is.